go.sum

```diff
@@ -9,45 +9,34 @@ cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxK
 cloud.google.com/go v0.45.1/go.mod h1:RpBamKRgapWJb87xiFSdk4g1CME7QZg3uwTez+TSTjc=
```

Comment:
I'm not gonna pretend to understand why removing context created such a large diff on the sum